scipy/signal/ltisys.py

```diff
@@ -2494,3 +2496,218 @@ def place_poles(A, B, poles, method="YT", rtol=1e-3, maxiter=30):
     full_state_feedback.nb_iter = nb_iter
 
     return full_state_feedback
+
+
+def controller_Hessenberg(A,B, C=None, tol=None):
```

Comment:
I have a new favorite term, thanks @WarrenWeckesser. +1 for lower case even though it's a proper name, esp. if that's what's done elsewhere in `scipy`